Source/Resources/Content/MSGraph/DeviceManagement/DeviceConfigurations/UX - Windows Hello Disabled.json

```diff
@@ -12,6 +12,6 @@
       }
     }
   ],
-  "description": "This policy disables Windows Hello. Windows Hello is enabled by default on any Azure AD Joined devices. This will prompt any Azure AD user to create a PIN at logon and is undesirable.",
+  "description": "What is it: Disables Windows Hello. \n Simeon Notes: Windows Hello is enabled by default on any Azure AD Joined devices. One of the effects of having Windows Hello is the users is prompted to create a PIN at logon, Simeon's experience has found that the majority of business do not want this option turned on.",
```

Comment:
**What does this do?**
Disables Windows Hello, a biometric sign-in system for Windows 10 that uses a near infrared (IR) imaging camera or fingerprint reader instead of a password to identify users.

**Why should you use this?**
If you want to disable Windows Hello and instead use passwords which are more in line with industry best practices.  

**What is the end user impact?**
Users will not be prompted to sign in with a biometric system.

**Learn more**
NA